docker-compose-parts/webapp.devel.yml

```
@@ -5,3 +5,4 @@ services:
       - "8081:8080"
     volumes:
       - ${BUILD_ROOT_PATH}/webapp/webapp:/isard:rw
+    command: sleep 999999999
```

I don't like this sleep but if it is needed will be better use infinite.
```suggestion
    command: sleep infinity
```